spring-cloud-dataflow-shell/src/main/java/org/springframework/cloud/dataflow/shell/command/ModuleCommands.java

```diff
@@ -191,14 +208,48 @@ public Object getValue(int row, int column) {
 				int currentRow = row - 1;
 				if (mappings.get(key).size() > currentRow) {
 					return mappings.get(key).get(currentRow);
-				} else {
+				}
+				else {
 					return null;
 				}
 			}
 		};
 		return DataFlowTables.applyStyle(new TableBuilder(model)).build();
 	}
 
+	@CliCommand(value = IMPORT_MODULES, help = "Register all modules listed in a properties file")
+	public String importFromResource(
+			@CliOption(mandatory = true,
+					key = {"", "uri"},
+					help = "URI for the properties file")
+			String uri,
+			@CliOption(key = "local",
+				help = "whether to resolve the URI locally (as opposed to on the server)",
+				specifiedDefaultValue = "true",
+				unspecifiedDefaultValue = "true")
```

Comment:
I think it should be TRUE by default (whether specified or not). That just seems more intuitive when pointing to a location *in a shell command*... that you'd expect that location to be resolved by the shell process itself.